src/old_ui/Card/Card.jsx

```diff
@@ -1,9 +1,48 @@
 import cs from 'classnames'
+import PropTypes from 'prop-types'
```

Comment:
The actual refactor went like this:

1. Copy ui/Card into old_ui/Card
2. Add an `old` variant with the styles from the old old_ui/Card
3. Handle className prop which the old old_ui/Card used
4. Update imports ui/Card -> old_ui/Card
5. Remove ui/Card

Reply:
I.e., big green diff is not new code - just copied from ui/Card

Reply:
sure

Reply:
also it's blue =P

![Screenshot 2024-04-24 at 13 07 05](https://github.com/codecov/gazebo/assets/105234307/2e4bae97-96f2-4ec3-a9b2-aa99b4423921)